src/types/Checkbox.ts

```diff
@@ -0,0 +1,5 @@
+export type PropsCheckbox =  {
```

Comment:
Fix props type according to the changes in your component.

Please ensure that the `onChange` prop is properly typed to handle an event object, specifically typed as React.ChangeEvent<HTMLInputElement>.

Additionally, consider renaming the props interface from `PropsCheckbox` to `ICheckbox `if using an interface, or `CheckboxPropsType` if using a type.

Reply:
fixed